src/plugins/user-activity-events/client.js

```diff
@@ -23,12 +23,17 @@ function throttle(fn) {
 }
 
 module.exports = function recordUserEvents(client) {
+
 	if (document.addEventListener) {
-		document.addEventListener('click', throttle(function() {
-			client.sendEvent('userIsActive');
-		}));
-		document.addEventListener('keydown', throttle(function() {
+		var listener = throttle(function userActivityListener() {

```

Comment:
You don't need the function names here (or in the removal below). So this could just be:
```
var listener = throttle(function() {
    client.sendEvent('userIsActive');
});
```